x-pack/test/functional/apps/reporting_management/report_delete_pagination.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default ({ getPageObjects, getService }: FtrProviderContext) => {
+  const pageObjects = getPageObjects(['common', 'reporting']);
+  const log = getService('log');
+  const retry = getService('retry');
+  const security = getService('security');
+
+  const testSubjects = getService('testSubjects');
+  const esArchiver = getService('esArchiver');
+
+  describe('Delete reports', function() {
+    before(async () => {
+      await security.testUser.setRoles(['global_discover_read', 'reporting_user']);
+      await esArchiver.load('empty_kibana');
+      await esArchiver.load('reporting/archived_reports');
+      await pageObjects.common.navigateToActualUrl('kibana', '/management/kibana/reporting');
+      await testSubjects.existOrFail('reportJobListing', { timeout: 200000 });
+    });
+
+    after(async () => {
+      await esArchiver.unload('empty_kibana');
+      await esArchiver.unload('reporting/archived_reports');
+      await security.testUser.restoreDefaults();
+    });
+
+    it('Confirm report deletion works', async () => {
+      log.debug('Checking for reports.');
+
+      await retry.try(async () => {
+        await testSubjects.click('checkboxSelectRow-k9a9xlwl0gpe1457b10rraq3');
```

Comment:
Is this generated id `k9a9xlwl0gpe1457b10rraq3` always remains the same?

It is a bit confusing to understand what test is doing exactly, how about using page objects style:

```
await pageObjects.reporting.selectAllReports();
await pageObjects.reporting.clickDelete();
```

Reply:
```Does this generated id k9a9xlwl0gpe1457b10rraq3 always remain the same?``` 
Yes, the id of the report will always be same as its loaded from static data set via esArchiver. 
So either selecting one or selecting all will be the same as the main functionality am testing is if the report is deleted or not . Let me know if this logic is right.